packages/form-control/index.ts

```diff
@@ -0,0 +1 @@
+export * from './src';
```

Comment:
do we want to barrel the types through the index also? I would think that that might be helpful so that types would be available without having to import the types file itself?

Reply:
They should be, `src/index.ts` exports all from types.